hs_core/hydro_realtime_signal_processor.py

```diff
@@ -59,15 +59,57 @@ def handle_save(self, sender, instance, **kwargs):
         elif isinstance(instance, ResourceAccess):
             # automatically a BaseResource; just call the routine on it. 
             newinstance = instance.resource
-            newsender = BaseResource
-            self.handle_save(newsender, newinstance)
+            self.handle_save(BaseResource, newinstance)
 
+        elif isinstance(instance, CoreMetaData): 
+            try: 
+                newinstance = BaseResource.objects.get(metadata=instance)
+                self.handle_save(BaseResource, newinstance)
+            except Exception: 
+                pass
```

Comment:
I'd like to see something logged here as the only scenario I see this happening is if the metadata doesn't have a resource.

Reply:
Logged.